Utilities/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:16.04
```

Comment:
Should this go in a subdirectory to keep the script etc together not loose in Utilities?

Reply:
Yeah lets put these files in its own dir

Reply:
The `Dockerfile` and downloaded tarballs now go inside the `Utilities/Docker` folder. By the way, the tarballs need to be downloaded there because the Dockerfile `COPY` command doesn't support copying files above in the folder hierarchy.